source/_docs/configuration.markdown

```diff
@@ -26,6 +26,8 @@ _If you use Home Assistant Operating System, you can find `configuration.yaml` i
 
 _If you use Home Assistant Core, you can find `configuration.yaml` in the config folder passed to the `hass` command (default is `~/.homeassistant`)._
 
+_If you mount the Home Assistant Operating System SD Card on your computer, you can find `/supervisor/homeassistant/configuration.yaml` in the `hassos-data` partition._
```

Comment:
I don't think that this is in context here. This isn't really a normal route to edit config files?

Reply:
This is not a common route to edit config files. It's useful to rescue your system when other routes fail. The three preceding lines are about advanced routes too. That's why I believe this is where people would look for such an information (as I did).

Reply:
> It's useful to rescue your system when other routes fail. 

It should not be able to get in such situation, and thus you are documenting a workaround to a bug. Did you report this issue in which you can get into this situation?